compute/rn/release-information/release-notes-22-06-update1.adoc

```diff
@@ -0,0 +1,53 @@
+== 22.06 Update 1 Release Notes
+
+The following table outlines the release particulars:
+
+[cols="1,4"]
+|===
+|Build
+|22.06.197
+
+|Code name
+|Kepler, 22.06 Update 1
+//Tentative date
+|Release date
+|Jul 27, 2022
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|5aa618314e176d03e559e58d2eba50959365cdc145cba99f5d47d90737d233bf
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, fixes, and performance enhancements
+//GithubIssue PCSUP-issue
+// #39791 #PCSUP-10025 On-prem
+* Fixed the broken pipe error when https://prisma.pan.dev/api/cloud/cwpp/images#operation/get-images-download[downloading a large image CSV]. The error was fixed by extending the HTTP client timeout value.
+// #40032 #PCSUP-10097 On-prem only
+* Fixed the welcome tour screen for new users.
+// #39668 #PCSUP-9482
+* Fixed an issue wherein the Defenders blocked application deployments on SELinux due to incorrect SELinux labeling on proxy `runc`. The issue was fixed by applying the original runc's SELinux label to the created `runc` proxy binary.
+// #39821 On-prem only
+* Fixed https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/configure/certificates[Self-signed certificate] validity period error. The value is extended and should be a whole number greater than or equal to 1.
+// #39434 PCSUP-9587 + 1749
+* Fixed an error in evaluating the custom checks for registry with CRI-O Defender. Custom checks are now run on the materialized image path.
+// #39351 PCSUP-9555
+* Fixed error that overwrote the communication port after upgrading a Defender with a custom port from the Prisma Cloud Console UI.
+
+=== Known issues
+
+// #
+
+=== End of Support Notifications
+
+// #40122
+* Debian 9 (Stretch) has reached End of Life (EOL), and users of Debian 9 will stop receiving any CVE security vulnerabilities from the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-intelligence-stream-notifications/notifications/intelligence-stream-significant-impact[IS] feed associated with this OS version.
+
+=== Supported host Operating systems
+
+A comprehensive list of supported operating systems is found in our https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements].
```

Comment:
For a comprehensive list of supported operating systems, refer to the https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements].